src/arm/am572x-bone-common-univ.dtsi

```diff
@@ -0,0 +1,2505 @@
+/*
+ * Copyright (C) 2020 Deepak Khatri <deepaklorkhatri7@gmail.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+&dra7_pmx_core {
+    /************************/
+	/* P8 Header */
+	/************************/
+
+	/* P8_01                GND */
+
+	/* P8_02                GND */
+
+
+	/* P8_03  (ball AB8) gpio1_24 */
+	P8_03_default_pin: pinmux_P8_03_default_pin { pinctrl-single,pins = <
+		DRA7XX_CORE_IOPAD(0x379C, PIN_OUTPUT_PULLDOWN | INPUT_EN | MUX_MODE14) >; };	/* mmc3_dat6.gpio1_24 */	
```

Comment:
Shouldn't this use the new macro?

Reply:
If you are happy with them I will implement them everywhere they can fit :)

Reply:
The new macro seems handy. I just don't want to define the address entry in multiple locations. One place to be right or wrong and get fixed.

Reply:
Okay, I will gradually shift things to use those macros.